packages/core/src/lib/cookie.ts

```diff
@@ -159,8 +159,17 @@ export class SessionStore {
    * The JWT Session or database Session ID
    * constructed from the cookie chunks.
    */
-  get value() {
-    return Object.values(this.#chunks)?.join("")
+   get value() {
+    // Sort the chunks by their keys before joining
+    const sortedKeys = Object.keys(this.#chunks).sort((a, b) => {
+      const aSuffix = parseInt(a.split(".")[1] || "0");
```

Comment:
Just to make sure, we always take the last part here.
`const aSuffix = parseInt(a.split(".").pop() || "0");`

Reply:
You have an answer here.